app/services/final_renewal_reminder_service.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class FinalRenewalReminderService < RenewalReminderTextServiceBase
```

Comment:
I found during current work across multiple communications services that it is helpful to have the communications type in the service name - so maybe "FinalRenewalReminderTextService"